src/services/product.ts

```diff
@@ -32,3 +32,28 @@ export const purchaseAndUseProduct = async (
     }
   })
 }
+
+export const purchaseAndUseProductWithStartDate = async (
```

Comment:
Can we go with different naming? I thought that this is the same product but with an additional `startDate` parameter at a first glance.

How about we:
1. rename `purchaseAndUseProduct` to `purchaseAndUseListingProduct`
2. call this `purchaseAndUseGlobalProduct` or `purchaseAndUseDealerProduct`?

WDYT?

Reply:
ok, it is similar to my first approach -> https://github.com/carforyou/carforyou-api-client-pkg/pull/155/commits/3a3af3c31d1352e476f33798e8f500e85b887823

I will rename them to `purchaseAndUseListingProduct` and `purchaseAndUseDealerProduct`